### PR TITLE
Issue 372

### PR DESCRIPTION
**Pull request by naphelps**
References Issue #372 

Changes:

- Added the environmental variable $ENVSUBST_CONFIG to the Dockerfile for controlling the behavior of the utility envsubt.

- Changed the default UBI metadata labels for release to be the same as version, and vendor to Open Horizon.

- Moved around and rewrote README documentation for config/exchange-api.tmpl giving it its own subsection under Building

and Running the Docker Container.

### CODE REVIEW

**Comment by naphelps** [comment]
No application changes.

File: build.sbt [hunk 1]
```diff
@@ -28,17 +28,17 @@ val versionFunc = () => {
...
-        release              := "4.1.0", 
-        resolvers            += Classpaths.typesafeReleases, 
-        scalaVersion         := "2.12.10",     // tried updating to scala 2.13.1, but got many compile errors in intellij related to JavaConverters being deprecated
-        summary              := "'Open Horizon exchange-api image'", 
-        vendor               := "IBM", 
-        version              := versionFunc(), 
+        description                   := "'Containerized exchange-api'",
+        name                          := "amd64_exchange-api",
+        organization                  := "com.horizon",
+        release                       := versionFunc(),
```
Comment (sf2ne) [inline build.sbt:34]: I'm probably just out of the loop on this, but we want the `release` to just be whatever version the exchange is at? Isn't that going to get confusing?

Comment (naphelps) [reply build.sbt:34]: As far as I am aware Open Horizon as a whole does not have a version, so the only other thing we have is the version of the Exchange.

Comment (sf2ne) [reply build.sbt:34]: Ah ok, I understand now, thanks!

**Pull request merged**